LineRipple/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import MaterialComponent from '../MaterialComponent';
+import { VNode } from 'preact';
+
+export default class LineRipple extends MaterialComponent<JSX.HTMLAttributes, {}> {}
```

Comment:
Perhaps it would be an idea to define the MDCLineRipple (property `MDComponent`) too (see https://github.com/prateekbh/preact-material-components/blob/master/Select/index.d.ts#L23 for how to define it and https://github.com/material-components/material-components-web/tree/master/packages/mdc-line-ripple#mdclineripple for the definition of the class).

Reply:
oohhhh I didnt know about this